core-bundle/src/DependencyInjection/Configuration.php

```diff
@@ -326,6 +326,38 @@ static function (array $value): array {
                     ->prototype('scalar')->end()
                     ->defaultValue(['jpg', 'jpeg', 'gif', 'png', 'tif', 'tiff', 'bmp', 'svg', 'svgz', 'webp'])
                 ->end()
+                ->arrayNode('preview')
+                    ->addDefaultsIfNotSet()
+                    ->children()
+                        ->scalarNode('target_dir')
+                            ->info('The target directory for the cached previews.')
+                            ->example('%kernel.project_dir%/assets/previews')
```

Comment:
Why can't we reuse the `assets/images` directory?

Reply:
We could, but letting two different services handle the same directory to store image files is not a good idea IMO. It might also make sense to mount assets/previews somewhere else for performance or storage reasons. I think a clear distinction here is better than reusing the assets/images folder.

Reply:
Not sure about that.

First of all, I would prefer to have all generated images in the same folder. Storing thumbnails in `assets/images` and previews in `assets/previews` will confuse people, because they have to know the difference in order to find a file.

Second of all, `assets/images` is optimized in terms of storage and can handle a large number of generated files. It might even be mounted somewhere else already. So why rebuild everything a second time?

Reply:
How about keeping it how it is now but change the default for `preview.target_dir` to `assets/images/previews` ?
```suggestion
                            ->example('%kernel.project_dir%/assets/images/previews')
```

Reply:
Do you mean storing all the files in one single folder or do you mean `assets/images/previews/0`, `assets/images/previews/1` and so on?

Reply:
yes, `assets/images/previews/0`, `assets/images/previews/1` and so on.

Reply:
I will change the default to `assets/images/previews`

Reply:
As briefly discussed, `assets/images/deferred` should be moved to `var/deferred-images` in Contao 5, therefore it makes sense to store the previews in `assets/previews` so we can have a "clean" `assets/images` directory in Contao 5 again.